src/view/pages/Farmer/tabs/Info/index.tsx

```diff
@@ -1,48 +1,34 @@
 import * as React from 'react';
-import { Container, Header, H1, H2, H3, Content, Grid, Row, Col, Form, Item, Input, Label, Radio, Button, Right, ListItem, InputGroup, Picker }
-from 'native-base';
-import { View, Text, StyleSheet } from 'react-native';
 
 import ViewFarmer from './viewFarmer';
 import EditFarmer from './editFarmer';
 import AddFarmer from './addFarmer';
-
-import styles from './style';
 import createPage from '../../../../generators/Page/index';
 
-interface OwnState {
+interface OwnStateType {
   mode?: string;
 }
 
-export interface OwnProps {
-  // farmerFirstName: string;
-  // farmerLastName: string;
-  // farmerPhoneNumber: string;
-  // farmerBusinessName: string;
-  // farmerNotes: string;
-  // selectedPaymentCycle: string;
-  // selectedPaymentMethod: string;
-  // modeHandler?: Function;
-}
-
+/**
+ * Modes to determine how to render farmer information
+ */
 export const modes = {
-  add: 'add',
   edit: 'edit',
   view: 'view',
 };
 
 /**
  * Container for application
  */
-class FarmerInformation extends React.Component<OwnProps, OwnState> {
+class FarmerInformation extends React.Component<{}, OwnStateType> {
```

Comment:
Please keep the PropsType just to make it more maintainable, as this is a page component and may well be receiving injected props in the future (such as from a fab button). In addition, James will be hooking this up to Redux, so maybe define the whole template Bea sent in `developer`

```
interface OwnPropsType {
}

interface DispatchPropsType {
}

interface StorePropsType {
}

type PropsType = OwnPropsType & DispatchPropsType & StorePropsType;

interface OwnStateType {
}
```

Reply:
Also please rename to match the naming scheme used by Bea for her tabs